viz-lib/src/lib/value-format.tsx

```diff
@@ -5,16 +5,24 @@ import numeral from "numeral";
 import { isString, isArray, isUndefined, isFinite, isNil, toString } from "lodash";
 import { visualizationsSettings } from "@/visualizations/visualizationsSettings";
 
+
 numeral.options.scalePercentBy100 = false;
 
 // eslint-disable-next-line
 const urlPattern = /(^|[\s\n]|<br\/?>)((?:https?|ftp):\/\/[\-A-Z0-9+\u0026\u2019@#\/%?=()~_|!:,.;]*[\-A-Z0-9+\u0026@#\/%=~()_|])/gi;
 
 const hasOwnProperty = Object.prototype.hasOwnProperty;
 
+function NullValueComponent() {
+  return <span className="text-muted"><i>{visualizationsSettings.nullValue}</i></span>;
```

Comment:
Instead of using the `<i>` tag, I think we should define a class `.display-as-null` in `viz-lib/src/visualizations/table/renderer.less`

Reply:
@eradman 
Thank you very much for the review !

I just changed the code to create `.display-as-null` class in `lib/src/visualizations/table/renderer.less` instead of `<i>` tag.

https://github.com/getredash/redash/pull/7439/commits/baabbdf1d836a3d473572ccc2b60370d0000c739